Spigot-API-Patches/0026-Add-command-to-reload-permissions.yml.patch

```diff
@@ -0,0 +1,113 @@
+From b70b7cc1d6b7ec3296c4401c0c42c9769e2fac4e Mon Sep 17 00:00:00 2001
+From: William <admin@domnian.com>
+Date: Fri, 18 Mar 2016 03:28:07 -0400
+Subject: [PATCH] Add command to reload permissions.yml
+
+https://github.com/PaperMC/Paper/issues/49
+
+diff --git a/src/main/java/org/bukkit/Bukkit.java b/src/main/java/org/bukkit/Bukkit.java
+index f93ca2e..cd73bfa 100644
+--- a/src/main/java/org/bukkit/Bukkit.java
++++ b/src/main/java/org/bukkit/Bukkit.java
+@@ -1188,4 +1188,11 @@ public final class Bukkit {
+     {
+         return server.spigot();
+     }
++
++    // Paper start
++    public static void reloadPermissions() {
++        server.reloadPermissions();
++    }
++    // Paper end
++
+ }
+diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
+index 91bde81..6b1f2a4 100644
+--- a/src/main/java/org/bukkit/Server.java
++++ b/src/main/java/org/bukkit/Server.java
+@@ -1018,4 +1018,6 @@ public interface Server extends PluginMessageRecipient {
+     }
+ 
+     Spigot spigot();
++
++    void reloadPermissions(); // Paper
+ }
+diff --git a/src/main/java/org/bukkit/command/defaults/ReloadCommand.java b/src/main/java/org/bukkit/command/defaults/ReloadCommand.java
+index c70d512..f91063b 100644
+--- a/src/main/java/org/bukkit/command/defaults/ReloadCommand.java
++++ b/src/main/java/org/bukkit/command/defaults/ReloadCommand.java
+@@ -1,25 +1,40 @@
+ package org.bukkit.command.defaults;
+ 
++import java.util.ArrayList;
+ import java.util.Arrays;
++import java.util.Collections;
```

Comment:
Is the synchronized block necessary? Can't we just catch bad calls with a
sync catcher?
-- 
Joseph Hirschfeld
joe@ibj.io
Ichbinjoe
Skype: ichbinjoehirschfeld